classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php

```diff
@@ -391,7 +399,7 @@ protected function _saveEditorDecision($args, $request, $formName, $redirectOp =
 			$notificationMgr->updateNotification(
 				$request,
 				$notificationTypes,
-				array($submission->getUserId()),
+				$authorUserIds,
```

Comment:
Again, this might send notifications to users that are not direct part of this review process, I think.